src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -66,6 +66,10 @@ workbenchActionsRegistry.registerWorkbenchAction(new SyncActionDescriptor(Toggle
 if (isWindows || isLinux) {
 	workbenchActionsRegistry.registerWorkbenchAction(new SyncActionDescriptor(ToggleMenuBarAction, ToggleMenuBarAction.ID, ToggleMenuBarAction.LABEL), 'View: Toggle Menu Bar', viewCategory);
 }
+workbenchActionsRegistry.registerWorkbenchAction(new SyncActionDescriptor(NavigateUpAction, NavigateUpAction.ID, NavigateUpAction.LABEL, null), 'View: Navigate Up', viewCategory);
```

Comment:
I understand that the command name might not be verbose enough.
Do you mean that the ```View: Navigate Up``` should be renamed as ```Navigate to next View to the top``` or to change the class name itself? Something like ```class NavigateUp``` to ```class NavigateViewToTheTop```?